recipes/shillelagh/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "shillelagh" %}
+{% set version = "0.8.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/shillelagh-{{ version }}.tar.gz
+  sha256: 5be1af7441b66c32bdd6be20dd9f7c3d70dd04dab8831a7e5041d42fe059a409
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - shillelagh = shillelagh.console:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - pyscaffold >=3.2a0,<3.3a0
+    - python >=3.7
+  run:
+    - apsw >=3.9.2,<4
+    - google-auth >=1.23.0,<2
+    - importlib-metadata
+    - prompt_toolkit >=3,<4
+    - pygments >=2.8,<3
+    - python >=3.7
+    - python-dateutil >=2.8.1,<3
+    - pytz ==2021.1
+    - pyyaml >=5.4,<6
+    - requests >=2.25.1,<3
+    - requests-cache ==0.5.2
+    - sqlalchemy >=1.3,<2
+    - sphinx >=4.0.1,<5
+    - tabulate ==0.8.9
+    - typing-extensions <4,>=3.7.4.3
+    - typing_extensions >=3.7.4.3,<4
+
+test:
+  imports:
+    - shillelagh
+    - shillelagh.adapters
```

Comment:
These are both empty. Maybe `shillelagh.console` to get the equivalent import to running the console script?